package.json

```diff
@@ -42,7 +42,7 @@
     "@typescript-eslint/parser": "^4.6.1",
     "applicationinsights": "^1.7.5",
     "browserify": "^17.0.0",
-    "downlevel-dts": "^0.7.0",
+    "@standardlabs/downlevel-dts": "^0.7.1",
```

Comment:
This is to pick up the changes I've suggested to upstream here: https://github.com/sandersn/downlevel-dts/pull/48